qbittorrent/umbrel-app.yml

```diff
@@ -0,0 +1,30 @@
+manifestVersion: 1
+id: qbittorrent
+category: Networking
+name: qBittorrent
+version: "4.5.2"
+tagline: A fast, easy and free BitTorrent client
+description: >-
+  The qBittorrent project aims to provide an open-source software alternative to µTorrent.
+  
+  Additionally, qBittorrent runs and provides the same features on all major platforms (FreeBSD, Linux, macOS, OS/2, Windows).
+  
+  Bittorrent is based on the Qt toolkit and libtorrent-rasterbar library.
+  
+  ⚠️ qBittorrent downloads torrents over the Clearnet, not Tor.
+developer: qBittorrent
+website: https://qbittorrent.org/
+dependencies: []
+repo: https://github.com/qBittorrent/qBittorrent
+support: https://github.com/qBittorrent/qBittorrent/issues
```

Comment:
I also see this forum we could use here: https://forum.qbittorrent.org/. Have you come across other support networks for qBittorent?

Reply:
In my opinion I think it might be best to stick with the Github issues page for this. Transmission has it set to their Github issues page too.